akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingRememberEntitiesSpec.scala

```diff
@@ -41,7 +41,8 @@ object ClusterShardingRememberEntitiesSpec {
   }
 
   val extractShardId: ShardRegion.ExtractShardId = msg ⇒ msg match {
-    case id: Int ⇒ id.toString
+    case id: Int                     ⇒ id.toString
+    case ShardRegion.StartEntity(id) ⇒ id
```

Comment:
Yes ofc. not sure what I was thinking when I didn't think of that there are extractShardIds in all the tests.. :(